content/docs/concepts/traffic-management/routing/_index.md

```diff
@@ -90,7 +90,7 @@ and specify the [routing rules](/docs/concepts/traffic-management/routing/virtua
 
 The advantage of this configuration method is that it decouples the application
 code from the evolution of its dependent services. This in turn provides
-monitoring benefits. For details, see [Mixer policies and telemetry](/docs/concepts/policies-and-telemetry/).
+monitoring benefits. For details, see [Mixer policies and telemetry](/docs/reference/config/policy-and-telemetry/).
```

Comment:
This should point to the new observability concepts doc. That said, this file doesn't exist in the master branch yet, because the traffic managements concepts rewrite has been reverted due to the hugo issue.
```suggestion
monitoring benefits. For details, see [Observability](/docs/reference/config/observability/).
```